src/ui/components/CreateIdentifier/components/IdentifierStage3.tsx

```diff
@@ -128,8 +155,7 @@ const IdentifierStage3 = ({
         customClass="identifier-stage-3"
         primaryButtonText={`${i18n.t("createidentifier.confirm.continue")}`}
         primaryButtonAction={() => {
-          // @TODO - sdisalvo: send the request
-          //  console.log(state);
+          createMultisigIdentifier();
```

Comment:
Isn't that at line 39 already enough @iFergal ?

Reply:
Nope =p That's not how promises work in JavaScript.

If I call an async function, it returns a promise and it's up to me as the caller to await that promise or not. Otherwise it wouldn't be async if there wasn't a choice, it would just be blocking sync.

The calling function has to also await too

Reply:
this still needs to be awaited